README.md

```diff
@@ -298,6 +298,40 @@ module.exports = {
 }
 ```
 
+- use the `import` or `require` syntax to directly import the resolver object:
+
+```js
+// .eslintrc.mjs
+import myResolver from '../../../my-resolver'
```

Comment:
Ideally, we could use `eslint-import-resolver-typescript`, `eslint-import-resolver-vite`, and `eslint-import-resolver-oxc` as an example! What do you think?